### ISSUE TITLE
Unacceptable behaviour on Search

### ISSUE THREAD

**Issue by yashrajbothra** [openemr/openemr#3141]
### Describe the bug

Search doesnt work with Enter work with click 

### To Reproduce
https://www.loom.com/share/7894fdcb2730452d93dbae067380bed4

### Client configuration

- Browser:Chrome
- Operating system:Win 10

**Comment by yashrajbothra** [comment]
Also on Global Setting

**Comment by skshahnawaz** [comment]
@bradymiller Please assign this issue to me. I am working on this

**Comment by bradymiller** [comment]
@yashrajbothra , did you want to do this or can I assign it to @skshahnawaz ?

**Comment by yashrajbothra** [comment]
No, Problem for me. Please go for it @bradymiller

**Comment by bradymiller** [comment]
@skshahnawaz , it is all yours :)

**Comment by skshahnawaz** [comment]
Thanks, will fix this soon

**Comment by mdsupport** [comment]
@skshahnawaz 
Originally search worked to highlight navigation labels as well actual setting including the description.  Somewhere during the refactors only tabs started getting highlighted.  That leaves user to figure out specific hit on a tab.  

At the time of that feature, bootstrap was not available.  Now we should transition to 'bg-warning' class and remove all search specific hard coding.  If you need assistance, perhaps @tywrenn can provide help.

If possible, please look to restore original behavior.  Test by searching for `restful`.  At least following should be highlighted :  
1. Connectors tab
2. 'Enable OpenEMR REST API' label
3. Checkbox (if possible - let bootstrap do its thing)

**Comment by bradymiller** [comment]
Note the search does highlight the items now (not sure if it looks at descriptions); this was recently fixed. I think this is just pointing out issue of when focus on entering in a search term and then hitting enter (in that case, it submits the globals rather than performing search). Agree bg-warning would be nice.

**Comment by mdsupport** [comment]
> this was recently fixed
Great.

Another nice to have behavior for someone lazy would be to move focus to first hit.

**Comment by yashrajbothra** [comment]
@skshahnawaz Are you working on this issue?

**Comment by skshahnawaz** [comment]
Yes I am working on it. Exploring a few things. Will keep you updated

**Comment by tywrenn** [comment]
Update: The highlight was using the `<mark>` element

**Comment by kojiromike** [comment]
Search works with Enter in master ~ 7.0.4. Not sure when this was resolved.
